src/java.base/windows/native/libjava/java_props_md.c

```diff
@@ -128,15 +128,19 @@ getEncodingInternal(LCID lcid)
     return ret;
 }
 
-static char* getConsoleEncoding()
+static char* getConsoleEncoding(boolean output)
```

Comment:
I assume it would be BOOL but I see several other functions here are using boolean with TRUE/FALSE so it should be okay.

Reply:
Oh that's right. Changed boolean to BOOL.